arbitrum-docs/how-arbitrum-works/06-optimistic-rollup.mdx

```diff
@@ -0,0 +1,299 @@
+  - A special case of adding your bond to a successor RBlock is when you create a new RBlock as a successor to an RBlock you’re already bonded on.
+- If you’re bonded only on the latest confirmed RBlock (and possibly earlier RBlocks), you or anyone else can ask to have your bond refunded. Your bonded funds will be returned to you, and you will no longer be a bonder.
+- If you lose a challenge, your bond is removed from all RBlocks and you forfeit your bonded funds.
+
+Notice that once you are bonded on an RBlock, there is no way to unbond. You are committed to that RBlock. Eventually one of two things will happen: that RBlock will be confirmed, or you will lose your bond. The only way to get your bond back is to wait until all of the RBlocks you are bonded on are confirmed.
+
+#### Setting the current minimum bond amount
+
+One detail we deferred earlier is how the current minimum bond amount is set. Normally, this is just equal to the base bond amount, which is a parameter of the Nitro chain. However, if the chain has been slow to make progress in confirming RBlocks, the bond requirement will escalate temporarily. Specifically, the base bond amount is multiplied by a factor that is exponential in the time since the deadline of the first unresolved RBlock passed. This ensures that if malicious parties are placing false bonds to try to delay progress (despite the fact that they’re losing those bonds), the bond requirement goes up so that the cost of such a delay attack increases exponentially. As RBlock resolution starts advancing again, the bond requirement will go back down.
+
+### Rules for Confirming or Rejecting RBlocks
+
+The rules for resolving RBlocks are fairly simple.
+
+The first unresolved RBlock can be confirmed if:
+
+- the RBlock's predecessor is the latest confirmed RBlock, and
+- the RBlock's deadline has passed, and
+- there is at least one bonder, and
+- All bonders are bonded to the RBlock.
+
+The first unresolved RBlock can be rejected if:
+
+- the RBlock's predecessor has been rejected, or
+- all of the following are true:
+  - the RBlock's deadline has passed, and
+  - there is at least one bonder, and
+  - no bonder is bonded on the RBlock.
+
+A consequence of these rules is that once the first unresolved RBlock's deadline has passed (and assuming there is at least one bonder bonded on something other than the latest confirmed RBlock), the only way the RBlock can be unresolvable is if at least one bonder is bonded on it and at least one bonder is bonded on a different RBlock with the same predecessor. If this happens, the two bonders are disagreeing about which RBlock is correct. It’s time for a challenge, to resolve the disagreement.
+
+<!-- -->
+
+<!-- fraud proofs/challenges -->
+
+## Validators
```

Comment:
I'm not sure about this section here. It looks like a good introduction (or concept page) for validators that could live in the "Run a node" section, instead of here. But let's leave it there for now.